docs/source/resources/define-metadata.rst

```diff
@@ -816,6 +816,38 @@ pinned down to the build string level. This will
 supersede any dynamic or compatible pinning that
 conda-build may otherwise be doing.
 
+Whitelisting shared libraries
+-----------------------------
+
+The ``missing_dso_whitelist`` build key is a list of globs for
+dynamic shared object (DSO) files that should be ignored when
+examining linkage information.
+
+During the post-build phase, the shared libraries in the newly created
+package are examined for linkages which are not provided by the
+package's requirements or a predefined list of libraries. If such
```

Comment:
> or a predefined list of libraries

Well, this is the whitelist itself!

Reply:
Maybe 'a predefined list of system libraries' would be better, because conda-build does have that concept too and yup, it a DSO is in either list (in code the fixed whitelist is called `whitelist`, but I may change it to `sysroot_whitelist` to disambiguate that from `missing_dso_whitelist`).